core/src/main/java/brooklyn/catalog/internal/BasicBrooklynCatalog.java

```diff
@@ -986,7 +1045,12 @@ private DeploymentPlan makePlanFromYaml(String yaml) {
     }
     
     private CatalogItem<?,?> addItemDto(CatalogItemDtoAbstract<?, ?> itemDto, boolean forceUpdate) {
-        checkItemNotExists(itemDto, forceUpdate);
+        CatalogItem<?, ?> existingDto = checkItemIsDuplicateOrDisallowed(itemDto, true, forceUpdate);
+        if (existingDto!=null) {
+            // it's a duplicate, and not forced, just return it
+            log.trace("Using existing duplicate for catalog item {}", itemDto.getId());
```

Comment:
it could get called a lot in hot backup.  and we verified that what we would do would be no-op before we did nothing so not a useful log message i think.